services/postfix/aliases.nix

```diff
@@ -0,0 +1,500 @@
+{tld}:
+{
+#
+# System, committee and important email aliases.
+#
+# $Id: aliases,v 1.18 2015/11/18 19:16:50 koffee Exp koffee $
+#
+
+#
+# NOTES
+# =====
+# - For committee position aliases during the changeover period, list the
```

Comment:
all the positions should me mailing lists not aliases to avoid this

Reply:
I disagree. Positions can be versioned here. Mailing lists come with a lot of overhead, and I don't trust mailman's archive security very much.

Reply:
Fine then plz group all positions emails together under this comment